AttributeError: 'Namespace' object has no attribute 'version'

### Actual Behavior

Running in the terminal:

`conda skeleton cran SuperLearner`

results in the following error message:

`Traceback (most recent call last):
  File "/opt/anaconda/anaconda3/bin/conda-skeleton", line 11, in <module>
    sys.exit(main())
  File "/opt/anaconda/anaconda3/lib/python3.6/site-packages/conda_build/cli/main_skeleton.py", line 66, in main
    return execute(sys.argv[1:])
  File "/opt/anaconda/anaconda3/lib/python3.6/site-packages/conda_build/cli/main_skeleton.py", line 62, in execute
    version=args.version, config=config)
AttributeError: 'Namespace' object has no attribute 'version'`

### Expected Behavior

Based on this answer to a previous conda-build issue, found here: 

https://github.com/conda/conda/issues/5624

it seems that possible to me this fails because of the description file for the SuperLearner package, found here:

https://cran.r-project.org/web/packages/SuperLearner/DESCRIPTION

Namely the error suggests to me (and I am probably completely wrong) that the command fails because the package description has a field which was unexpected by the build script. The expected behavior for me would have been to continue installing regardless of any unexpected fields in the description, e.g. by ignoring those fields, or something else.

Since the SuperLearner package is not in Anaconda's R package repository, this is supposed to be the only way to install it (within the conda framework, i.e. to get dependency tracking and version control, etc. all of the reasons one would want to have a package management system).

https://github.com/conda/conda/issues/4355

So the expected behavior would be that the command suggested for installing CRAN packages works more readily.

**UPDATE:** Trying to circumvent this by jankily running ```install.packages("SuperLearner")``` from within R (which again is *not* preferred due to the lack of package management system inherent to such an approach), it seems that a problem occur firsts with installing the dependencies, which perhaps also causes an issue when trying to create a conda skeleton. The first roadblock was installing `gtools`, which is in the conda R repository. Then the command from within R almost works except for the dependcy `nnls`. This appears to have a gfortran dependency, which would relate to a previous conda-build GitHub issue (involving conda and R packages with Fortran dependencies): https://github.com/conda/conda-build/issues/2642

```
* installing *source* package ‘nnls’ ...
** package ‘nnls’ successfully unpacked and MD5 sums checked
** libs
/opt/anaconda/anaconda3/bin/x86_64-apple-darwin13.4.0-gfortran   -fPIC  -march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -I/opt/anaconda/anaconda3/include -L/opt/anaconda/anaconda3/lib  -c lawson_hanson_nnls.f -o lawson_hanson_nnls.o
make: /opt/anaconda/anaconda3/bin/x86_64-apple-darwin13.4.0-gfortran: No such file or directory
make: *** [lawson_hanson_nnls.o] Error 1
ERROR: compilation failed for package ‘nnls’
* removing ‘/opt/anaconda/anaconda3/lib/R/library/nnls’
```

### Steps to Reproduce

<!-- Steps to reproduce issue. -->

On a Mac OS X with an Anaconda installation and the R package channel for conda installed, type

`conda skeleton cran SuperLearner`

into the terminal.

##### `Output of conda info`
<!-- Between the ticks below, paste the output of 'conda info' -->
```
    active environment : None
 populated config files : 
          conda version : 4.4.11
    conda-build version : 3.7.1
         python version : 3.6.4.final.0
       base environment : /opt/anaconda/anaconda3  (writable)
           channel URLs : https://repo.continuum.io/pkgs/main/osx-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/osx-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/osx-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/osx-64
                          https://repo.continuum.io/pkgs/pro/noarch
          package cache : /opt/anaconda/anaconda3/pkgs
       envs directories : /opt/anaconda/anaconda3/envs
               platform : osx-64
             user-agent : conda/4.4.11 requests/2.18.4 CPython/3.6.4 Darwin/15.6.0 OSX/10.11.6
                UID:GID : 504:20
             netrc file : None
           offline mode : False


```

--- a/conda_build/skeletons/cran.py
+++ b/conda_build/skeletons/cran.py
@@ -286,17 +286,14 @@ def add_parser(repos):
     cran.add_argument(
         "--add-maintainer",
         help="Add this github username as a maintainer if not already present.",
-        default=argparse.SUPPRESS,
     )
     cran.add_argument(
         "--version",
         help="Version to use. Applies to all packages.",
-        default=argparse.SUPPRESS,
     )
     cran.add_argument(
         "--git-tag",
         help="Git tag to use for GitHub recipes.",
-        default=argparse.SUPPRESS,
     )
     cran.add_argument(
         "--all-urls",
@@ -318,7 +315,6 @@ def add_parser(repos):
         "--use-binaries-ver",
         help=("Repackage binaries from version provided by argument instead of building "
               "from source."),
-        default=argparse.SUPPRESS,
     )
     cran.add_argument(
         "--use-noarch-generic",

